src/widgets/Notifications.php

```diff
@@ -130,7 +130,7 @@ public static function getCountUnseen(){
         $userId = Yii::$app->getUser()->getId();
         $count = NotificationModel::find()
             ->andWhere(['or', 'user_id = 0', 'user_id = :user_id'], [':user_id' => $userId])
-            ->andWhere(['<=', 'sent_at', date('Y-m-d H:i:s')])
+            ->andWhere(['<=', 'send_at', date('Y-m-d H:i:s')])
```

Comment:
why this change? if you count the unseen notifications you want to count only the onces already sent so it's ok to use 'sent_at' attribute